src/test/java/org/openrewrite/java/dependencies/UpgradeDependencyVersionTest.java

```diff
@@ -91,6 +91,62 @@ void upgradeGuavaInGradleProject() {
         );
     }
 
+    @Test
+    void upgradeMockitoInGradleProject() {
+        rewriteRun(recipeSpec -> {
+              recipeSpec.beforeRecipe(withToolingApi())
+                .recipe(new UpgradeDependencyVersion("org.mockito", "*", "4.x", null, null, null));
+          },
+          buildGradle(
+            """
+              plugins {
+                id 'java-library'
+              }
+
+              repositories {
+                mavenCentral()
+              }
+
+              dependencies {
+                testImplementation("org.mockito:mockito-junit-jupiter:3.12.4")
+              }
+              """,
+            """
+              plugins {
+                id 'java-library'
+              }
+
+              repositories {
+                mavenCentral()
+              }
+
+              dependencies {
+                testImplementation("org.mockito:mockito-junit-jupiter:4.11.0")
```

Comment:
Has mockito completely stopped putting out 4.x releases? I'd think so given that [4.11 stems from December 2022](https://github.com/mockito/mockito/releases/tag/v4.11.0), but figured to check since we wouldn't want to update the test output if there's a newer release.